apps/examples/src/examples/speech-bubble/SpeechBubble/helpers.tsx

```diff
@@ -234,28 +181,13 @@ export function intersectLineSegmentLineSegment(
 /**
  * Inverse linear interpolation
  */
-export function invLerp(a: number, b: number, v: number) {
+function invLerp(a: number, b: number, v: number) {
 	return (v - a) / (b - a)
 }
 /**
  * Maps a value from one range to another.
  * e.g. mapRange(10, 20, 50, 100, 15) => 75
  */
-export function mapRange(a1: number, b1: number, a2: number, b2: number, s: number) {
+function mapRange(a1: number, b1: number, a2: number, b2: number, s: number) {
 	return lerp(a2, b2, invLerp(a1, b1, s))
 }
-
-export const getHandleinShapeSpace = (shape: SpeechBubbleShape): SpeechBubbleShape => {
-	const newShape = deepCopy(shape)
-	newShape.props.handles.handle.x = newShape.props.handles.handle.x * newShape.props.w
-	newShape.props.handles.handle.y = newShape.props.handles.handle.y * newShape.props.h
-	return newShape
-}
-
-export const getHandlesInHandleSpace = (shape: SpeechBubbleShape): SpeechBubbleShape => {
-	const newShape = deepCopy(shape)
-	newShape.props.handles.handle.x = newShape.props.handles.handle.x / newShape.props.w
-	newShape.props.handles.handle.y = newShape.props.handles.handle.y / newShape.props.h
-
-	return newShape
-}
```

Comment:
Wrong direction here. The names are misleading for both functions (returning a shape and not handles), and can be easily confused between them. As mentioned above, it's also confusing to switch between normalized and not normalized values within the same object (shape).

Reply:
This makes a lot of sense, thanks